plugins/scenario/hu.bme.mit.gamma.scenario.statechart.generator/xtend-gen/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
Remove `*.statechart.generator/xtend-gen/hu/bme/mit/gamma/scenario/statechart/.Generator.java._trace` from the index.